src/logic/operate.js

```diff
@@ -0,0 +1,25 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const nOne = Big(numberOne);
+  const nTwo = Big(numberTwo);
+
+  switch (operation) {
+    case '-':
+      return nOne.minus(nTwo).toString();
+    case '+':
+      return (nOne + nTwo);
+    case 'X':
+      return nOne.times(nTwo).toString();
+    case '÷':
+      if (nTwo === '0') {
+        return '0';
+      }
```

Comment:
Return `'Infinity'` or `division by zero error`
or something else linked to error